civihr_employee_portal/src/Security/PublicFirewall.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Drupal\civihr_employee_portal\Security;
+
+/**
+ * This class is responsible for checking if anonymous users can access
+ * certain routes
+ */
+class PublicFirewall {
+
+  /**
+   * Anonymous users can access these paths
+   *
+   * @var array
+   */
+  public static $publicRoutes = [
+    '/^welcome-page$/',
+    '/^sites\/default\/files\/logo.jpg$/', // if logo is missing
+    '/^request_new_account\/ajax$/', // from login page
+    '/^user((?!\/register).)*$/', // user* except user/register
+    '/^yoti-connect*/' // yoti login plugin
+  ];
+
+  /**
+   * @param \stdClass $user the user to check access for
```

Comment:
Please add a description to this method and also remember that in Drupal code standard, the param description should be in a newline